src/EnergyPlus/WindowManager.cc

```diff
@@ -877,9 +874,8 @@ namespace WindowManager {
                 }
                 if (Material(LayPtr).GlassSpectralAndAngle) {
                     if (!BGFlag) AllGlassIsSpectralAverage = false;
-                    numptDAT = TableLookup(PerfCurve(Material(LayPtr).GlassSpecAngTransDataPtr).TableIndex).NumX2Vars;
+                    numptDAT = wle.size();
```

Comment:
I am not sure why we should use fixed wavelength. We may give a user to input their values.

Reply:
The users input curves or tables that are used to define behavior at all wavelengths. We are not limiting the users here. Instead, the users are now less constrained in how they define the spectral and angular performance. Also, if `wlt` and `wle` are not the same dimensions, the spectral/visible average calculations will not work properly.